client/jest.config.js

```diff
@@ -0,0 +1,52 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  collectCoverageFrom: [
+    '**/src/components/**/*.{js}',
```

Comment:
```suggestion
    '**/app/components/**/*.{js,jsx}',
```
I don't think we have any folder with name "src", instead it is "app" folder. :)